src/components/room-list.module.scss

```diff
@@ -1,3 +1,58 @@
 .container {
+  height: 570px;
+  padding-bottom: 15px;
   display: flex;
+  flex-direction: column;
+  background-color: white;
 }
+  h2{
```

Comment:
you should follow convention about code style

it is not okay that in one file you have spacebar between `{` and className and in another places in your file you don't have it